src/collections/members/hauwa-ismail-ahmad/index.mdx

```diff
@@ -0,0 +1,14 @@
+---
+name: Hauwa Ismail Ahmad
+position: Contributor
+image_path: ./hauwa-ismail-ahmad.webp
+github:
+twitter: hauwy_ismyl
+linkedin: hauwa-ismail-75224a255
+layer5:
```

Comment:
Here empty github user handle, cloud user id and location is causing the error, you are facing in preview!